mvp/README.md

```diff
@@ -0,0 +1,19 @@
+
+**User-Centric** design enables the continuous delivery of value to users. This means that putting a good, functional, tool in the hands of users quickly is more valuable than delivering a perfect tool when it's too late.
+
+Design Thinking accomplishes this by establishing the boundaries of the product early through the definition of the **Minimum Viable Product**, or **MVP**. Once the MVP is delivered, the team continues to iterate on the solution, incrementally improving and enhancing it by leveraging continuous feedback from users and stakeholders.
+
+The **Minimum Viable Product** is a product version delivered with just enough features to satisfy early adopters and elicit feedback for future development.  The Minimum Viable Product should satisfy the following conditions:
+1. Has sufficient **_value_** to **_attract_** early adopters
+2. Demonstrates enough **_value_** to **_retain_** these users
+3. Establishes a **_feedback_** loop to **_guide_** future development
+
+Eric Ries has [defined](https://www.agilealliance.org/glossary/mvp) the MVP as "that version of a new product which allows a team to collect the maximum amount of validated learning about customers with the least effort."
+
+There are three factors to be considered when establishing the MVP:
+1. Desirability - what does the user need?
+2. Viability - what is is the business case for the product?
+3. Feasibility - what is technically & financially feasible?
+
+In Design Thinking, human-centered design begins with Desirability.  The convergence of user need,  business viability, and technical & financial feasibility outlines the solution.
+
```

Comment:
MVP (minimum viable products) - 
- consider needs of users/business
- user friendly/ease of use/ and adaptable to changes
- financially  feasible